airflow/contrib/operators/qubole_operator.py

```diff
@@ -174,6 +175,9 @@ def get_hook(self):
         # Reinitiating the hook, as some template fields might have changed
         return QuboleHook(*self.args, **self.kwargs)
 
+    def get_extra_links(self, dttm, link_name):
```

Comment:
docstring

Reply:
This is an implementation of https://github.com/apache/airflow/pull/3533/files#diff-664e0133fa172b6e0060f2dd7b2c3504R2948